docs/src/js/base/combobox.js

```diff
@@ -106,18 +106,18 @@ function filter(widget) {
 
     if (numChars === 0) {
         widget._listboxWidget._activeDescendant.reset();
-        widget._listboxWidget.items.forEach(el => el.hidden = false);
+        widget._listboxWidget.items.forEach(el => (el.hidden = false));
     } else {
-        const matchedItems = Util.nodeListToArray(widget._listboxWidget.items).filter((el) => {
+        const matchedItems = Util.nodeListToArray(widget._listboxWidget.items).filter(function(el) {
```

Comment:
How come are we going away from arrow functions? I think eslint can support arrow function syntax which I think we should prefer going forward.

Reply:
I actually changed this one to satisfy some other rule around formatting of arrow functions, I forget which rule now. I agree that the arrow function here was very readable. Anyhow, I think it's good to be comfortable with either syntax. I think the stylelint rules around it are a bit lacking.

Reply:
The rule in question is `arrow-body-style`. This one basically forces you to write arrow functions without return and brackets if it's only a single line
I personally don't like the single line return to always be enabled because sometimes (like this case) you have a long statement which looks better in brackets.
Maybe we can disable that rule and change it to arrow function?

Reply:
That seems reasonable to me. Thanks.